core/src/com/ericc/the/game/map/Map.java

```diff
@@ -1,31 +1,37 @@
 package com.ericc.the.game.map;
 
+import com.badlogic.ashley.core.Entity;
 import com.badlogic.gdx.math.GridPoint2;
 import com.badlogic.gdx.math.MathUtils;
 import com.ericc.the.game.Mappers;
 import com.ericc.the.game.Media;
 import com.ericc.the.game.TileTextureIndicator;
 import com.ericc.the.game.components.PositionComponent;
-import com.ericc.the.game.components.StaircaseDestinationComponent;
-import com.ericc.the.game.entities.Stairs;
 import com.ericc.the.game.helpers.FogOfWar;
 import com.ericc.the.game.utils.RectangularBitset;
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.HashSet;
 
 public class Map {
 
+    private final GridPoint2 tmpGridPoint2 = new GridPoint2();
```

Comment:
komentarz?

Reply:
albo jeszcze lepiej: zmiana nazwy zamiast komentarza

Reply:
Odciążanie garbage collectora.

Reply:
@agluszak To jest jedyna sensowna nazwa. To jest po prostu tymczasowa zmienna tego typu, ona nie ma żadnego znaczenia semantycznego. Jest używana do tego, żeby móc przekazywać GridPoint2 do funkcji, które tego oczekują, bez tworzenia nowego GridPoint2 za każdym razem.

Reply:
powtórzę jeszcze raz to co wyżej napisałem: bez sensu że my w tę i nazad zmieniamy inty na GridPointy i GridPointy na inty. Gdybyśmy używali wszędzie GridPointów to nie byłoby problemu.